geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/executor/pubsub/SubCommandExecutor.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *
+ */
+
+
+package org.apache.geode.redis.internal.executor.pubsub;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.apache.geode.redis.internal.executor.Executor;
+import org.apache.geode.redis.internal.executor.RedisResponse;
+import org.apache.geode.redis.internal.netty.Command;
+import org.apache.geode.redis.internal.netty.ExecutionHandlerContext;
+
+public class SubCommandExecutor implements Executor {
```

Comment:
Since there are a lot of commands in Redis that have sub commands, I think it would make more sense to follow the pattern of our other executors and name this after the primary command, so `PubsubExecutor` (or `PubSubExecutor`?) (also, all related test classes would need to be renamed).  Check out `ClusterExecutor`, which is named after the primary command, but has a case switch for sub commands.

Reply:
Right - it's unfortunate we already have those `PubSub*` classes. Perhaps even naming these with `PubsubSub*`?

Reply:
yeah, I went back and forth about this.  I had all these files prefaced with pubsub initially, but "pubsubsubcommand*" feels a little like a tongue-twister. ..:).  My reasoning to leave it as subcommand* is that all of these files that start with subcommand live within directories named pubsub, which seems to clarify what command they belong to.   I think there is a little thought that needs to happen to break up the things currently called pubsub into Publish and subscribe.  I'd rather not do all that within this commit, if you all are ok with that, but will make a point of doing it before this "module" is done.  let me know if you all can live with that...  (so, to clarify, the idea I'm pitching is to leave the naming as it is for now,  but before the module is completed, rename stuff currently named pubsub  so that name becomes available, and then look at what to name all this subcommand stuff )

Reply:
Yes, we should definitely fix the other classes now that this command is introduced. I would like, at least, for this PR to have the executor named consistently as the other commands: `PubsubExecutor`